debug.js

```diff
@@ -0,0 +1,45 @@
+const spawn         = require('child_process').spawn;
+const webdriver     = require('selenium-webdriver');
+const _             = require('lodash/fp');
+
+const os = process.platform  === 'win32' ? 'win' : process.platform;
+
+const fileToDebug = process.argv.splice(2)[0];
```

Comment:
Could we include a step to check if the file exists, and throw an error, just in case of typos?